### DISCUSSION TITLE
Suport for use in HoloViz Panel apps

### DISCUSSION THREAD

**Discussion by MarcSkovMadsen** [tqdm/tqdm#1147] [Show and tell]
FYI.

Just wanted to share another use case.

I am a contributor to and user of [HoloViz Panel](https://panel.holoviz.org/). Panel is a framework for analytics apps using the tools you know and love. I also run https://awesome-panel.org which show cases Panel apps.

`tqdm` is one of the tools I know and love and I want it supported in Panel. So I have started a PR https://discourse.holoviz.org/t/tqdm-indicator-pr/1976

Feel free to check it out and throw a few hints my way on the implementation if you are interested.

https://user-images.githubusercontent.com/42288570/111037171-4ae25200-8423-11eb-82f1-058adc4862fd.mp4

https://user-images.githubusercontent.com/42288570/111037175-4fa70600-8423-11eb-84d1-1c93e7d0d7b8.mp4

**Comment by casperdcl** [comment]
Sure - you mean something based on https://github.com/holoviz/panel/pull/2079 to be baked into `tqdm` itself similar to `tqdm.notebook`, `tqdm.tk`, `tqdm.contrib.telegram` and so on? Sure; feel free to open a PR here adding something like `tqdm.contrib.panel`.

**Comment by MarcSkovMadsen** [comment]
Thanks. I actually just want to share knowledge about the integration. :-)

And now I even Got feedback on the name.Thanks.
